nerdlandbot/commands/notify.py

```diff
@@ -37,15 +37,24 @@ async def act_subscribe(self, ctx: commands.Context, list_name: str, user_id: in
             msg = translate("all_sub_success", await culture(ctx))
             return await ctx.send(msg)
         else:
-            # Error if list does not exist
-            if not guild_data.does_list_exist(list_name):
+            # Check if 1 or more lists with list_name exist
+            list_matches = guild_data.get_filtered_lists(list_name)
+
+            # No matches => Error
+            if not list_matches:
                 msg = translate("list_err_does_not_exit", await culture(ctx))
                 return await ctx.send(msg)
 
-            # Subscribe user and error if failed
-            if not await guild_data.sub_user(list_name, user_id):
-                msg = translate("list_err_already_subscribed", await culture(ctx)).format(str(user_id), list_name)
-                return await ctx.send(msg)
+            # Single match => Sub
+            elif len(list_matches) == 1:
+                # Subscribe user and error if failed
+                if not await guild_data.sub_user(list_name, user_id):
```

Comment:
Using the list_name here will use the filter keyword that the user entered. Instead the key for the list_matches should be used